src/sentry/incidents/endpoints/serializers.py

```diff
@@ -315,7 +322,55 @@ def validate(self, data):
         This includes ensuring there is either 1 or 2 triggers, which each have actions, and have proper thresholds set.
         The critical trigger should both alert and resolve 'after' the warning trigger (whether that means > or < the value depends on threshold type).
         """
-        data["aggregate"] = aggregation_function_translations[data.pop("aggregation")]
+        if "aggregate" in data and "aggregation" in data:
+            # `aggregate` takes precedence over `aggregation`, so just drop `aggregation`
+            # if both are present.
+            data.pop("aggregation")
+        if "aggregation" in data:
+            data["aggregate"] = aggregation_function_translations[data.pop("aggregation")]
+        if "aggregate" in data:
+            try:
+                snuba_filter = build_snuba_filter(
+                    # TODO: Stop hardcoding this once we support multiple datasets
+                    QueryDatasets.EVENTS,
```

Comment:
This will come from a parameter called `dataset`

Reply:
👍 Will implement this separately.